docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -581,7 +581,14 @@ Returns file level QC information. It will return a JSON object of the form:
         "File" : $Filename
     },
     "QC" : "Pass|Fail",
-    "Selected" : boolean
+    "Selected" : boolean,
+    "Caveats" : {
```

Comment:
If this is Caveat*s* shouldn't it be an array, not an object?

Reply:
right. It should be an array of objects... Updating the doc since the code already does it this way.